docs/how-tos/integrating-with-contentful.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 6
+---
+
+# Integrating with Contentful
+
+It is common for marketing and content teams to want an A/B testing solution with a point-and-click visual editor to make straightforward copy or image optimizations. However, these solutions most often rely on blocking scripts that cause performance degradations—slowing your site down by 200-500 milliseconds in best-case scenarios. You could avoid these performance issues by fully implementing feature flags for marketing experiments—but this will mean marketing has to rely on engineering for every test they want to run.
+
+Using a headless CMS like Contentful, and a lightweight, robust feature flagging SDK, like Eppo, allows the best of both worlds: an easy way for marketing teams to run tests without sacrificing performance. In this tutorial, we will walk through how to set up Eppo’s Node SDK so Marketing teams can run their own experiments by updating [Content](https://www.contentful.com/help/content-model-and-content-type/) defined in Contentful and feature flags in Eppo. 
+
+
+## Setting up Contentful
+
+For this example, imagine we are running an experiment on the Eppo homepage. There are a few places where we might want to experiment on this page:
+
+1. The hero title (”Run reliable, impactful experiments”)
+2. The hero body (”Eppo streamlines …”)
+3. The CTA text (”Preview Eppo Now”), and the CTA link (”www.geteppo.com/get-access”)
+4. The demo video
+
+To build this experiment out in our Contentful space, create a new Content Model with the name “Homepage Baseline”. Add the following content fields:
+
+![Contentful field setup](/img/how-tos/integrating-with-contentful/contentful-field-setup.png)
+
+Now to create our variants, duplicate your “Homepage Baseline” Content Model and give it the name “Homepage Redesign”. Update the content fields in the “Homepage Redesign” Content Model with new language that you want to test:
+
+![Contentful variant setup](/img/how-tos/integrating-with-contentful/contentful-variant-setup.png)
+
+
+:::note
+For this tutorial, we are including a Control and Variant for an A/B test, but you can repeat this process to add additional variants.
+:::
+
+Now that we have the content defined, we’ll need to get the entry ID for our two pages. In Contentful, the entry ID can be found on the content page and by clicking on the 3 dots next to your Content Model and clicking ‘Copy entry ID’. Save these for now, we will use them in our Eppo setup next.
+
+![Getting Contentful entry ids](/img/how-tos/integrating-with-contentful/getting-contentful-entry-ids.png)
+
+## Setting up Eppo
+
+Next, we’ll create a [corresponding flag in Eppo](https://docs.geteppo.com/feature-flag-quickstart/) (Feature Flags >> Create). For each variant above, simply add a new variant in the Eppo UI. For each variant value, paste in the corresponding Contentful entry ID. Make sure to also save your Feature Flag key – you will need it for your Node implementation later.
```

Comment:
```suggestion
Next, we’ll create a [corresponding flag in Eppo](/feature-flag-quickstart/) (Feature Flags >> Create). For each variant above, simply add a new variant in the Eppo UI. For each variant value, paste in the corresponding Contentful entry ID. Make sure to also save your Feature Flag key – you will need it for your Node implementation later.
```

Reply:
docusaurus should add the base url when it builds the docs

Reply:
good catch